Automattic/jetpack
# Button styles: ensure connection button can accomodate longer text

Fixes #21710

#### Changes proposed in this Pull Request:

* The button didn't previously accomodate long text, which can be an issue in some languages.

**Before**

<img width="629" alt="Screenshot 2021-11-11 at 13 17 11" src="https://user-images.githubusercontent.com/426388/141297069-d5d04a08-2a6e-4768-a414-f68fa311725c.png">

**After**

![image](https://user-images.githubusercontent.com/426388/141318798-6bb3b7bc-f50d-4172-b64e-86eeeb29d828.png)

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

1. Create a brand new site.
2. Go to Settings > General and switch the language to French
3. Go to Tableau de Bord > Mises à jour and update translations
4. Go to the Jetpack menu, and start the connection process, but do not connect your user.
5. Load the Jetpack menu again, and click on the button to start the user connection process.
6. Ensure the connection button looks good.
